checkov/secrets/scan_git_history.py

```diff
@@ -1,16 +1,26 @@
 from __future__ import annotations
 
-import git
 import logging
+import os
 from typing import TYPE_CHECKING, Dict
 from detect_secrets.core import scan
 
 if TYPE_CHECKING:
     from detect_secrets import SecretsCollection
 
+os.environ["GIT_PYTHON_REFRESH"] = "quiet"
+try:
+    import git
+    git_import_error = None
+except ImportError as e:
+    git_import_error = e
```

Comment:
can we add here a log for the import error also, not only in the function? :)

Reply:
I don't want to add the same log twice. if we don't use the function, the warning is irrelevant.